packages/sheets-ui/src/facade/f-worksheet.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IRenderManagerService } from '@univerjs/engine-render';
+import { FWorksheet } from '@univerjs/sheets/facade';
+import { SheetSkeletonManagerService } from '../services/sheet-skeleton-manager.service';
+
+interface IFWorksheetSkeletonMixin {
+
+    /**
+     * Refresh the canvas.
+     */
+    refreshCanvas(): void;
+}
+
+class FWorksheetSkeletonMixin extends FWorksheet implements IFWorksheetSkeletonMixin {
+    override refreshCanvas(): void {
+        const renderManagerService = this._injector.get(IRenderManagerService);
+        const unitId = this._fWorkbook.id;
+        const render = renderManagerService.getRenderById(unitId);
+
+        if (!render) {
+            throw new Error('Render not found');
```

Comment:
Please consider to expose more debug info in error messages. For example, `unitId`.